azurerm/internal/services/network/network_connection_monitor_resource.go

```diff
@@ -814,16 +842,22 @@ func flattenNetworkConnectionMonitorEndpoint(input *[]network.ConnectionMonitorE
 			address = *item.Address
 		}
 
+		var endpointType network.EndpointType
```

Comment:
This should be a plain string rather than the aliased type

Reply:
updated